aspnetcore/fundamentals/logging/index.md

```diff
@@ -1,25 +1,59 @@
 ---
-title: Logging in ASP.NET Core
+title: Logging in .NET Core and ASP.NET Core
 author: tdykstra
-description: Learn about the logging framework in ASP.NET Core. Discover the built-in logging providers and learn more about popular third-party providers.
+description: Learn how to use the logging framework provided by the Microsoft.Extensions.Logging NuGet package.
+monikerRange: '>= aspnetcore-2.1'
 ms.author: tdykstra
 ms.custom: mvc
 ms.date: 07/11/2019
 uid: fundamentals/logging/index
 ---
-# Logging in ASP.NET Core
+# Logging in .NET Core and ASP.NET Core
 
-By [Steve Smith](https://ardalis.com/) and [Tom Dykstra](https://github.com/tdykstra)
+By [Tom Dykstra](https://github.com/tdykstra) and [Steve Smith](https://ardalis.com/)
 
-ASP.NET Core supports a logging API that works with a variety of built-in and third-party logging providers. This article shows how to use the logging API with built-in providers.
+.NET Core supports a logging API that works with a variety of built-in and third-party logging providers. This article shows how to use the logging API with built-in providers.
+
+::: moniker range=">= aspnetcore-3.0"
+
+Most of the code examples shown in this article are from ASP.NET apps. The logging-specific parts of these code snippets apply to any .NET app that uses the [Generic host](xref:fundamentals/host/generic-host). For information about how to use the Generic Host in non-web console apps, see [Hosted services](xref:fundamentals/host/hosted-services).
```

Comment:
We should be consistent with `.NET Core` and `ASP.NET Core` vs. `.NET` and `ASP.NET`, it keeps changing throughout the article.